openid-connect-server-webapp/src/main/resources/db/clients_oracle.sql

```diff
@@ -0,0 +1,51 @@
+--
```

Comment:
New file (not using `clients.sql`) because:
* Oracle doesn't support mass INSERTs
* Oracle doesn't have a BOOLEAN type
* Oracle's MERGE syntax is slightly different